pkg/tsdb/cloudwatch/metrics.go

```diff
@@ -368,6 +368,7 @@ var metricsMap = map[string][]string{
 	"AWS/Polly":                {"2XXCount", "4XXCount", "5XXCount", "RequestCharacters", "ResponseLatency"},
 	"AWS/PrivateLinkEndpoints": {"ActiveConnections", "BytesProcessed", "NewConnections", "PacketsDropped", "RstPacketsReceived"},
 	"AWS/PrivateLinkServices":  {"ActiveConnections", "BytesProcessed", "EndpointsCount", "NewConnections", "RstPacketsReceived"},
+	"AWS/Prometheus":           {"AlertManagerAlertsReceived", "AlertManagerNotificationsFailed", "AlertManagerNotificationsThrottled", "DiscardedSamples", "RuleEvaluations", "RuleEvaluationFailures", "RuleGroupIterationsMissed"},
```

Comment:
Where did `AlertManagerNotificationsFailed` come from? I didn't see it in the linked docs.

Reply:
Good catch and expected question. AWS doc is not always good and this is an example. `AlertManagerNotificationsFailed` is valid metrics (I requested also AWS to update their doc):
![image](https://user-images.githubusercontent.com/1182932/190222393-c71ffa71-fcb2-4f38-b6a9-4cad2003119e.png)